### ISSUE TITLE
False-positive in `effector/no-ambiguity-target`

### ISSUE THREAD

**Issue by igorkamyshev** [effector/eslint-plugin#57]
```ts
export const resetError = (
  event: Event<any>,
  error: Store<CardValidationError | null>,
) => {
  sample({
    clock: guard({
      clock: event,
      source: error,
      filter: (err) => err !== null,
    }),
    fn: () => null,
    target: error,
  });
};

```
